src/main/resources/TaskManagerCode/Administration.xml

```diff
@@ -20,31 +20,29 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc version="1.1">
-  <web>TaskManager</web>
+<xwikidoc>
```

Comment:
I exported TaskManagerCode with Admin tools app. I don't know if it makes any difference. I worked on XE7.0.
Either way, I will redo the PR and close this one for now.